traitlets/config/application.py

```diff
@@ -374,7 +374,6 @@ def print_options(self):
             return
         lines = ['Options']
         lines.append('=' * len(lines[0]))
-        lines.append('')
```

Comment:
(I'm not sure if this comment is still valid)

The pyrpose behind this changes was to reduce the total number of lines printed in the console, to avoid scrolling.
Additionally, I wanted to apply the "Law of Proximity" design principle, and place the section-body closer to its header, in comparisons to the preceding section-body.

Is thatt OK?